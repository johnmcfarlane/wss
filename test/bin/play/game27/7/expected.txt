 24  3,9  | WAY
 24  3,9  | WEY
 24  3,9  | WYE
 21  2,10 - YE
 21  2,10 - YO
 19  2,10 - OY
 19  2,2  | YA
 19  2,2  | YE
 18  1,3  - EYL
 18  3,9  | WAE
 18  3,9  | WEA
 18  3,9  | WEI
 18  3,9  | WOA
 18  3,9  | WOE
 16  1,1  - YIKE
 16  1,1  - YOKE
 15  3,4  - AQUAE
 15  1,1  - YAK
 15  1,1  - YOK
 14  4,1  | AY
 14  1,9  | CAY
 14  1,9  | COY
 13  2,10 - EO
 13  3,7  - FEASE
 13  2,10 - IO
 13  2,10 - OE
 12  1,3  - AIL
 12  3,7  - FEES
 12  3,7  - FEIS
 12  5,6  | GEE
 12  5,6  | GIE
 12  1,1  - OAKY
 12  1,3  - OIL
 11  7,2  | AE
 11  2,2  | OE
 11  7,2  | OE
 11  1,8  | YCIE
 10  4,1  | AE
 10  4,1  | AI
 10  2,1  - AKEY
 10  2,1  - IKEY
 10  4,1  | IO
 10  2,1  - OKAY
 10  2,1  - OKEY
 10  6,3  - ROEY
  9  1,1  - AIK
  9  3,1  - KAY
  9  3,1  - KEY
  9  3,1  - KYE
  9  1,1  - OAK
  9  1,1  - OIK
  9  6,5  - XI
  9  7,1  | YA
  9  7,1  | YE
  9  7,1  | YI
  9  5,1  | YO
  9  7,1  | YO
  8  1,8  | ACY
  8  1,9  | CAI
  8  1,9  | CIA
  8  1,9  | CIE
  8  1,9  | COE
  8  2,1  - EKOI
  8  1,8  | ICY
  8  2,1  - OKIA
  8  2,1  - OKIE
  8  2,9  | OY
  8  6,2  - PAY
  8  6,2  - POY
  8  6,2  - PYA
  8  6,2  - PYE
  8  6,3  - RYA
  8  6,3  - RYE
  8  2,2  - YO
  7  7,1  | AE
  7  7,1  | AI
  7  2,1  - AKE
  7  2,1  - AKO
  7  5,2  - APII
  7  5,2  - APIO
  7  5,1  | EO
  7  7,1  | EO
  7  2,1  - IKE
  7  5,1  | IO
  7  7,1  | IO
  7  3,1  - KAE
  7  3,1  - KAI
  7  3,1  - KEA
  7  3,1  - KIE
  7  3,1  - KOA
  7  3,1  - KOI
  7  7,1  | OE
  7  1,7  | OECI
  7  7,1  | OI
  7  2,1  - OKA
  7  2,1  - OKE
  7  5,2  - OPAI
  7  6,2  - PEAI
  7  6,7  - SOYA
  7  3,9  | WY
  6  1,8  | ACE
  6  5,2  - APE
  6  1,8  | ECO
  6  5,2  - EPA
  6  5,2  - EPI
  6  1,2  - IAO
  6  7,7  | IAO
  6  1,8  | ICE
  6  7,7  | III
  6  3,1  - KA
  6  3,1  - KI
  6  1,8  | OCA
  6  5,2  - OPA
  6  5,2  - OPE
  6  7,7  | OY
  6  6,2  - PEA
  6  6,2  - PIA
  6  6,2  - PIE
  6  6,2  - POA
  6  6,2  - POE
  6  6,2  - POI
  6  6,3  - RAY
  6  6,3  - ROY
  5  5,2  - OP
  5  6,2  - PA
  5  6,2  - PE
  5  6,2  - PI
  5  6,2  - PO
  5  6,7  - SAY
  5  6,7  - SOY
  5  6,7  - SYE
  5  3,9  | WE
  5  3,9  | WO
  4  2,2  - EO
  4  2,2  - IO
  4  7,7  | IO
  4  7,7  | OE
  4  2,9  | OE
  4  7,7  | OI
  4  2,9  | OI
  4  2,2  - OO
  4  2,9  | OO
  4  3,2  - OY
  4  6,3  - REA
  4  6,3  - RIA
  4  6,3  - RIE
  4  6,3  - RIO
  4  6,3  - ROE
  4  6,3  - ROI
  4  2,8  | YO
  3  2,3  - AL
  3  2,3  - EL
  3  6,3  - RE
  3  6,7  - SAE
  3  6,7  - SAI
  3  6,7  - SAO
  3  6,7  - SEA
  3  6,7  - SEI
  3  6,7  - SIA
  3  6,7  - SIE
  3  6,7  - SOE
  2  2,8  | EO
  2  2,8  | IO
  2  3,2  - OE
  2  3,2  - OI
  2  3,2  - OO
  2  2,8  | OO
  2  6,7  - SI
  2  6,7  - SO
