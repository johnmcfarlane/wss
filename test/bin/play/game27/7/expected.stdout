 24  3,9  | WAY
 24  3,9  | WYE
 21  2,10 - YE
 21  2,10 - YO
 19  2,2  | YA
 19  2,2  | YE
 18  3,9  | WAE
 18  3,9  | WOE
 16  1,1  - YOKE
 15  3,4  - AQUAE
 15  1,1  - YAK
 15  1,1  - YOK
 14  4,1  | AY
 14  1,9  | CAY
 14  1,9  | COY
 13  2,10 - EO
 13  3,7  - FEASE
 13  2,10 - OE
 12  1,3  - AIL
 12  3,7  - FEES
 12  5,6  | GEE
 12  1,1  - OAKY
 12  1,3  - OIL
 11  7,2  | AE
 11  2,2  | OE
 11  7,2  | OE
 10  4,1  | AE
 10  4,1  | AI
 10  2,1  - OKAY
  9  3,1  - KAY
  9  3,1  - KEY
  9  3,1  - KYE
  9  1,1  - OAK
  9  6,5  - XI
  9  7,1  | YA
  9  7,1  | YE
  9  5,1  | YI
  9  7,1  | YI
  9  5,1  | YO
  9  7,1  | YO
  8  1,8  | ICY
  8  2,9  | OY
  8  6,2  - PAY
  8  6,2  - PYA
  8  6,2  - PYE
  8  6,3  - RYA
  8  6,3  - RYE
  8  2,2  - YO
  7  7,1  | AE
  7  5,1  | AI
  7  7,1  | AI
  7  5,1  | EO
  7  7,1  | EO
  7  3,1  - KAE
  7  3,1  - KAI
  7  3,1  - KEA
  7  3,1  - KOA
  7  3,1  - KOI
  7  7,1  | OE
  7  5,1  | OI
  7  7,1  | OI
  7  2,1  - OKA
  7  2,1  - OKE
  7  6,7  - SOYA
  6  1,8  | ACE
  6  5,2  - APE
  6  5,2  - APO
  6  1,8  | ICE
  6  3,1  - KA
  6  3,1  - KI
  6  1,8  | OCA
  6  5,2  - OPE
  6  7,7  | OY
  6  6,2  - PEA
  6  6,2  - PIA
  6  6,2  - PIE
  6  6,2  - POI
  6  6,3  - RAY
  5  5,2  - IP
  5  5,2  - OP
  5  6,2  - PA
  5  6,2  - PE
  5  6,2  - PI
  5  6,2  - PO
  5  6,7  - SAY
  5  6,7  - SOY
  5  3,9  | WE
  5  3,9  | WO
  4  2,2  - EO
  4  7,7  | OE
  4  2,9  | OE
  4  7,7  | OI
  4  2,9  | OI
  4  2,2  - OO
  4  2,9  | OO
  4  3,2  - OY
  4  6,3  - RAI
  4  6,3  - RIA
  4  6,3  - ROE
  4  2,8  | YO
  3  2,3  - AL
  3  2,3  - EL
  3  6,3  - RE
  3  6,7  - SAE
  3  6,7  - SEA
  3  6,7  - SEI
  2  2,8  | EO
  2  3,2  - OE
  2  3,2  - OI
  2  3,2  - OO
  2  2,8  | OO
  2  6,7  - SI
  2  6,7  - SO
